config/core/roles/sources-controller-clusterroles.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: knative-eventing-source-controller
+  name: knative-eventing-sources-controller
```

Comment:
should that rename be covered in the release notes ? :eyes:

Reply:
it is a cluster role and internal